docs/node/web3.md

```diff
@@ -0,0 +1,201 @@
+---
+description: Web3 gateway for Emerald and Sapphire ParaTimes
+---
+
+# Oasis Web3 Gateway for your EVM ParaTime
+
+This guide will show you how to set up the Oasis Web3 gateway for
+EVM-compatible ParaTimes such as Emerald and Sapphire.
+
+:::caution
+
+Each ParaTime requires its own instance of the Web3 gateway!
+
+:::
+
+## Prerequisites
+
+### Hardware
+
+* CPU:
+  * Minimum: 2.0 GHz x86-64 CPU
+  * Recommended: 2.0 GHz x86-64 CPU with 2 cores/vCPUs
+* Memory:
+  * Minimum: 4 GB of ECC RAM
+  * Recommended: 8 GB of ECC RAM
+* Storage:
+  * Minimum: 300 GB of SSD or NVMe fast storage
+  * Recommended: 500 GB of SSD or NVMe fast storage
+
+:::info
+
+To put the figures above into perspective, the Web3 gateway for Emerald with
+PostgreSQL encountered **210 GB** of database growth in ~5 months between
+Nov 18, 2021 and Apr 11, 2022 (since the [Emerald Mainnet launch]).
+
+:::
+
+### PostgreSQL
+
+The Web3 gateway stores blockchain data in a
+[PostgreSQL](https://www.postgresql.org/) database version **13.3** or higher.
+Install it by following the instructions specific to your operating system. You
+will need to create a separate database for each ParaTime. We also suggest that
+you create and use a different username for connecting to the database for each
+ParaTime.
```

Comment:
```suggestion
Install it by following the instructions specific to your operating system.

Because each ParaTime requires its own instance of the Web3 gateway, you will have to create a separate database and a separate user for each Web3 gateway instance.
```

Lets promote good practices.